x-pack/legacy/plugins/console_extensions/spec/generated/security.put_privileges.json

```diff
@@ -14,6 +14,6 @@
     "patterns": [
       "_security/privilege/"
     ],
-    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-put-privileges.html"
+    "documentation": "TODO"
```

Comment:
Same as above, let's manually revert this for now to not be blocked on upstream

Reply:
5b2c3eb